src/calendar/types.ts

```diff
@@ -8,12 +8,14 @@ export enum CalendarAction {
   ChangedView,
   CloseEventDetail,
   CloseProjectDashboard,
+  CloseReservationPage,
```

Comment:
The semantics jumped out at me here.  "Page" seems like to broad of a container term for what this is.  How about "Form" instead?
```suggestion
  CloseReservationForm,
```